packages/block-library/src/column/edit.js

```diff
@@ -66,9 +65,16 @@ function ColumnEdit( {
 	};
 
 	const widthWithUnit = Number.isFinite( width ) ? width + '%' : width;
+	const shadowProps = useShadowProps( attributes );
+	const flexBasisStyle = widthWithUnit
+		? { flexBasis: widthWithUnit }
+		: undefined;
 	const blockProps = useBlockProps( {
 		className: classes,
-		style: widthWithUnit ? { flexBasis: widthWithUnit } : undefined,
+		style: {
+			...shadowProps.style,
```

Comment:
I guess this change is not required as `column` block styles are directly be applied at parent level not to the inner blocks. Same is the case for `columns`